assets/dns/daemonset.yaml

```diff
@@ -89,4 +89,6 @@ spec:
       maxSurge: 0
       # Note: The daemon controller rounds the percentage up
       # (unlike the deployment controller, which rounds down).
-      maxUnavailable: 10%
+      # 10% metric is intended for traffic serving applications
+      # on all nodes.  33% is reasonable for DNS.
+      maxUnavailable: 33%
```

Comment:
The dns daemonset should be 10%, and the node-resolver daemonset (defined in `desiredNodeResolverDaemonSet`) should be 33%, right?

Reply:
I see I forgot it in the config for node resolver, and I'll add it there.  Should it be for both DNS and nodeResolver daemonsets?

Reply:
DNS should be 10% to minimize disruption (especially in case the cluster admin has configured DNS pods not to run on every node).